packages/@react-aria/grid/src/useGridCell.ts

```diff
@@ -36,6 +36,8 @@ interface GridCellAria {
   gridCellProps: HTMLAttributes<HTMLElement>
 }
 
```

Comment:
Changes in this file are for "using Option Tab in /story/table--hideheader will take you through buttons, if you go through a few of them, and then hit the 'up arrow', it'll put focus back above where the last non-Option-tab stop was"